src/FluentAvalonia/Interop/Win32Interop.cs

```diff
@@ -63,6 +66,33 @@ public static extern BOOL TrackPopupMenu(HMENU hMenu, uint uFlags,
     // Below is adapted from TerraFX.Interop.Windows, MIT License
     public static delegate*<HWND, int, nint> GetWindowLongPtr => &GetWindowLongPtrW;
 
+    public static int GetSystemMetricsWithFallback(int nIndex, uint dpi)
+    {
+        try
+        {
+            return GetSystemMetricsForDpi(nIndex, dpi);
+        }
+        catch (EntryPointNotFoundException)
```

Comment:
It would be better to use a Windows version check rather than Try/Catch. Take a look at the `ApplyTheme` method further below, as I already have a helper class for this.

Reply:
@amwx I have used the utility class you provided for the check. However, I found an issue with the function: it returns `false` instead of `true` when the system version exactly matches the major, minor, and build.